_docs/setup/install-sidecar.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Installing Istio Sidecar
```

Comment:
We agreed to not using sidecar anymore, but rather proxy or Envoy?

Reply:
Other way around. No proxy. Use Envoy or Sidecar. And given that we have multiple sidecar technologies, it makes sense to keep this section generic for the moment.